ricecooker/utils/pipeline/transfer.py

```
@@ -237,6 +238,131 @@
+        return [GoogleHttpError]
+
+    @property
+    def drive_service(self):
+        try:
+            from google.oauth2.service_account import Credentials
+            from googleapiclient.discovery import build
+        except ImportError:
+            raise RuntimeError(
+                "Google Drive downloads require google-auth and google-api-python-client libraries\n"
+                + instructions
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)